python/pdstools/utils/cdh_utils.py

```diff
@@ -325,6 +349,14 @@ def getMatches(files_dir, target):
         match = [file for name in names if re.findall(name.casefold(), file.casefold())]
         if len(match) > 0:
             matches.append(match[0])
+    if len(matches) == 0:
+        raise FileNotFoundError(
+            """Couldn't find a file with default ADM table names. Please try feeding model and predictor files seperately. \n
+        
+        Or check out getMatches function at https://github.com/pegasystems/pega-datascientist-tools/blob/master/python/pdstools/utils/cdh_utils.py
```

Comment:
I'd rather not point to source code - would rather include those default names in the function docstring so they can be included in the docs

Reply:
I wrote a docstring and added the default names to getMatches function now. But in the app you won't be able to see the docstring. So it would be nice if we could point the user to somewhere.